pages/privacy.tsx

```diff
@@ -1,232 +1,128 @@
-          apply uniformly to Wallora desktop website & Wallora mobile applications
-        </p>
-        <div className="font-semibold ml-1 mt-8">What is Personal Information?</div>
-        <p
-          style={{
-            font: '100% Roboto,"Segoe UI",Helvetica;',
-            color: '#261919',
-          }}
-          className="ml-1 mt-8"
-        >
-          'Personal Information' means data that identifies you and is associated with a specific person. Personal
-          information does not include information that is anonymous, aggregated, or is no longer identifiable to a
-          specific person.
-        </p>
-        <div className="font-semibold ml-1 mt-8">How We Collect Your Personal Information</div>
-        <p
-          style={{
-            font: '100% Roboto,"Segoe UI",Helvetica;',
-            color: '#261919',
-          }}
-          className="ml-1 mt-8"
-        >
-          We collect many kinds of information in order to operate effectively and provide you the best products,
-          services and experiences we can. We collect information, including personal information, when you provide it
-          to us, such as when you register or open an account, sign in, pay a bill, or call us for support. We may also
-          automatically collect information when you use our sites and services. We also may get information from other
-          companies or 3rd parties. We automatically collect information in a variety of ways, including from web forms,
-          through technologies like cookies, web logging, and software on your computer or other device.
-        </p>
-        <p
-          style={{
-            font: '100% Roboto,"Segoe UI",Helvetica;',
-            color: '#261919',
-          }}
-          className="ml-1 mt-8"
-        >
-          The types of personal information we collect and share depend on the product or service you have with us such
-          as name, address, mailing address, telephone number, email ID, credit card number, cardholder name, card
-          expiration date, information about your mobile phone and any details that may have been voluntarily provide by
-          the user in connection with availing any of the services on Wallora. When you browse through Wallora branded
-          website or mobile applications, we may collect information regarding the domain and host from which you access
-          the internet, the Internet Protocol [IP] address of the computer or Internet service provider [ISP] you are
-          using, and anonymous site statistical data.
-        </p>
-        <div className="font-semibold ml-1 mt-8">How We Use Your Personal Information</div>
-        <p
-          style={{
-            font: '100% Roboto,"Segoe UI",Helvetica;',
-            color: '#261919',
-          }}
-          className="ml-1 mt-8"
-        >
+      <div className="m-96 mt-16 mb-16 font-sans">
+        <h1 className="text-4xl">Privacy Policy</h1>
```

Comment:
add text-4xl to h1 in global.css so that h1 looks the same wherever else it is used